help/desktop-app-install-guide.md

```diff
@@ -119,26 +119,59 @@ command:
 
 Get a peek at new features before they're released!
 
-#### macOS, Windows, and most Linux distros
+{start_tabs}
+
+{tab|most-systems}
+
+#### Offline installer *(for macOS, Windows, and most Linux distros)*
+
+!!! warn ""
+
+    The app will not update automatically. You can repeat these steps to upgrade
+    to future versions.
+
```

Comment:
@andersk Just to check, is this warning accurate?

Reply:
No. Beta releases auto-upgrade in the same way as stable releases.